htdocs/modules/system/themes/modern/modern.php

```
@@ -243,18 +243,18 @@ private function getModuleStats(&$tpl)
     {
         /** @var XoopsModuleHandler $module_handler */
         $module_handler = xoops_getHandler('module');
+
         $criteria = new CriteriaCompo();
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '247', column '25'). (undefined)

(MissingImport)